### PR TITLE
Feature/bar13

### PR DESCRIPTION
**Pull request by codestreamtest**
This is my second PR to test with!

### CODE REVIEW

File: .idea/beerfridge.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```
Comment (planteater) [inline .idea/beerfridge.iml:3]: Put the beer in the fridge.

File: .idea/codestream.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CodeStream">
```
Comment (planteater) [inline .idea/codestream.xml:3]: xml yo mamma

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
```
Comment (codestreamtest2) [inline .idea/misc.xml:3]: CS2 just a single comment.

Comment (codestreamtest2) [reply .idea/misc.xml:3]: CS start a review foo.

File: .idea/codestream.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    <option name="teamId" value="5eff892b30f0f81ed047f6da" />
+    <option name="webViewContext" value="{&quot;chatProviderAccess&quot;:&quot;strict&quot;,&quot;currentTeamId&quot;:&quot;5d0800eb4733c111405de5c5&quot;,&quot;currentStreamId&quot;:&quot;&quot;,&quot;isRepositioning&quot;:false,&quot;panelStack&quot;:[&quot;landing-redirect&quot;],&quot;hasFocus&quot;:true,&quot;channelFilter&quot;:&quot;all&quot;,&quot;channelsMuteAll&quot;:false,&quot;codemarkFileFilter&quot;:&quot;all&quot;,&quot;codemarkTypeFilter&quot;:&quot;all&quot;,&quot;codemarkTagFilter&quot;:&quot;all&quot;,&quot;codemarkBranchFilter&quot;:&quot;all&quot;,&quot;codemarkAuthorFilter&quot;:&quot;all&quot;,&quot;codemarksFileViewStyle&quot;:&quot;inline&quot;,&quot;codemarksShowArchived&quot;:false,&quot;codemarksShowResolved&quot;:false,&quot;showFeedbackSmiley&quot;:true,&quot;route&quot;:{&quot;name&quot;:&quot;login&quot;,&quot;params&quot;:{}},&quot;spatialViewShowPRComments&quot;:false}" />
```
Comment (planteater) [inline .idea/codestream.xml:5]: Going to approve this shizzle.

File: .idea/.gitignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```
Comment (planteater) [inline .idea/.gitignore:2]: Moar change request.

Comment (planteater) [reply .idea/.gitignore:2]: Replying to an existing thread from CS as dave275

File: freezer.js [hunk 6]
```diff
@@ -0,0 +1,154 @@
+"use strict";
+import { ReviewDiffContentProvider } from "providers/diffContentProvider";
+import { ExtensionContext } from "vscode";
+import { BaseAgentOptions, CodeStreamAgentConnection } from "./agent/agentConnection";
+import { CodeStreamSession } from "./api/session";
+import { Commands } from "./commands";
+import { Config, configuration, ConfigurationWillChangeEvent } from "./configuration";
+import { LiveShareController } from "./controllers/liveShareController";
+import { NotificationsController } from "./controllers/notificationsController";
+import { StatusBarController } from "./controllers/statusBarController";
```
Comment (planteater) [inline freezer.js:10]: Tres line here.

File: foo.js [hunk 7]
```diff
@@ -19,6 +19,13 @@ export interface InsertTextCommandArgs {
 	indentAfterInsert?: boolean;
 }
 
+export interface ShowReviewDiffCommandArgs {
+	reviewId: string;
+	checkpoint: CSReviewCheckpoint;
+	repoId: string;
+	path: string;
+}
```
Comment (planteater) [inline foo.js:27]: hey ho comment in a pr - single

File: freezer.js [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+		this._versionBuild = agentOptions.extension.build;
+		this._versionFormatted = agentOptions.extension.versionFormatted;
+		this._agent = new CodeStreamAgentConnection(context, agentOptions);
+
+		context.subscriptions.push((this._session = new CodeStreamSession(config.serverUrl)));
+		context.subscriptions.push((this._notifications = new NotificationsController()));
+		context.subscriptions.push((this._vsls = new LiveShareController()));
+		context.subscriptions.push((this._commands = new Commands()));
+		context.subscriptions.push((this._codeActions = new CodeStreamCodeActionProvider()));
+		context.subscriptions.push((this._codeLens = new CodemarkCodeLensProvider()));
```
Comment (planteater) [inline freezer.js:40]: 3rd? comment in a review

**Pull request closed without merging**